Description: ecore_con: remove last occurence of EMILE_SSLv3
 This fixes compilation with gnutls
Author: Vincent Torri <vincent dot torri at gmail dot com>
Applied-Upstream: 93f45eedcfcb066ae1a4829a0f559979f62e71ec
Last-Update: 2016-01-16
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/lib/emile/emile_cipher_gnutls.c
+++ b/src/lib/emile/emile_cipher_gnutls.c
@@ -384,7 +384,6 @@
       case EMILE_TLSv1:
          priority = "NORMAL:%VERIFY_ALLOW_X509_V1_CA_CRT:!VERS-SSL3.0";
          break;
-      case EMILE_SSLv3:
       default:
          return NULL;
      }
